frontend/src/styles/shelfy.css

```
@@ -1,50 +1,293 @@
+  
+  /* Shadows */
+  --sh-shadow-sm: 0 4px 10px rgba(0,0,0,0.04);
+  --sh-shadow-md: 0 8px 24px rgba(0,0,0,0.06);
+  --sh-shadow-lg: 0 16px 36px rgba(0,0,0,0.1);
+  --sh-shadow-hover: 0 20px 48px rgba(0,0,0,0.12);
 }
 
 * { box-sizing: border-box; margin: 0; padding: 0; }
-body { font-family: system-ui, -apple-system, sans-serif; background: var(--sh-surface); color: #111; }
+body { 
+  font-family: 'Outfit', system-ui, -apple-system, sans-serif; 
```

Stylelint (17.5.0)

[error] Unexpected quotes around "Outfit" (font-family-name-quotes)

(font-family-name-quotes)